Formula/heartbeat.rb

```diff
@@ -20,15 +21,16 @@ def install
 
     cd "src/github.com/elastic/beats/heartbeat" do
       system "make"
-      libexec.install "heartbeat"
+      (libexec/"bin").install "heartbeat"
+      libexec.install "_meta/kibana"
```

Comment:
does heartbeat just automatically find this somehow?

Reply:
The intention is for these to be on the filesystem so that a user can import them into Kibana (by setting `dashboards.enabled: true` in their `heartbeat.yml`); by default Heartbeat will look in the Heartbeat home (which we now set to `#{prefix}/libexec`).